plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -94,6 +94,45 @@ def build_selectors
       verification_templates(main_only: false).first.build_selectors
     end
 
+    def metadata
```

Comment:
prefer calling this deploy_metadata or something like that since `metadata` is already used for kubernetes metadata